lib/train/extras/command_wrapper.rb

```diff
@@ -81,6 +81,9 @@ def verify
       when /sudo: sorry, you must have a tty to run sudo/
         ["Sudo requires a TTY. Please see the README on how to configure "\
           "sudo to allow for non-interactive usage.", :sudo_no_tty]
+      when /sudo: a terminal is required to read the password; either use/
+        ["Sudo cannot prompt for password because there is no terminal. "\
+            "Please provide the sudo password directly", :sudo_lack_of_permission]
```

Comment:
This looks good, but the symbol makes no sense any more - let's go with `:sudo_missing_terminal`?

Reply:
Did the change please check